platform/mds/mds/src/main/java/org/motechproject/mds/service/impl/EntityServiceImpl.java

```diff
@@ -951,7 +951,7 @@ public SchemaHolder getSchema() {
         LOGGER.debug("Retrieving entities for processing");
 
         stopWatch.start();
-        List<Entity> entities = allEntities.retrieveAll();
+        List<Entity> entities = allEntities.getActualEntities();
```

Comment:
can you check in git history if this method was used earlier here ? If yes, why someone changed it.